piksi_tools/console/gui_utils.py

```diff
@@ -68,6 +68,7 @@ def plot_square_axes(plot, xnames, ynames):
     except:  # noqa
         import traceback
         traceback.print_exc()
+        raise
```

Comment:
this squaring fails very often in practice, why raise here?

Reply:
This is quite experimental PR, we can revert some parts as necessary if somekind of conclusion is reached.